api/src/test/java/org/openmrs/module/registrationapp/converter/RegistrationSummaryExtensionsGeneratorTest.java

```diff
@@ -73,22 +78,23 @@ public void generate_shouldGenerateRegSummaryFromRegApp() throws Exception {
     
     @Test(expected = IllegalArgumentException.class)
     public void generate_shouldThrowWhenNotRegAppConfig() throws Exception {
-    	InputStream inputStream = getClass().getClassLoader().getResourceAsStream("registration_app.json");    	
-    	List<AppDescriptor> appDescriptors = new ObjectMapper().readValue(inputStream, new TypeReference<List<AppDescriptor>>() {});
+    	AppDescriptor appDescriptor = new AppDescriptor("fooId", "fooDesc", null, null, null, null, 0);
+    	appDescriptor.setInstanceOf("not.a.reg.app");
     	
-    	AppDescriptor appDescriptor = appDescriptors.get(0);
-    	appDescriptor.setId("acme.sysadmin.manage");
     	List<Extension> extensions = RegistrationSummaryExtensionsGenerator.generate(appDescriptor);
```

Comment:
You actually don't need to instantiate `extensions` here. You won't make any verifications out of it.

Also please, if possible, keep the **setup / replay / verif** breakdown comments. They really help the reader find in one snap what it is that you were actually replaying for a test.

Reply:
Cool @mks-d.